engines/sword2/maketext.cpp

```diff
@@ -674,7 +674,7 @@ void Sword2Engine::initializeFontResourceFlags() {
 	else
 		textLine = (char *)fetchTextLine(textFile, 54) + 2;
 
-	_system->setWindowCaption(textLine);
+	_system->setWindowCaption(Common::U32String(textLine));
```

Comment:
I am wondering if this is correct there. Do we know what encoding might be used in the resource? Assuming ASCII may not be right and maybe we need proper conversion?

By the way here is the title I get for the French version: `Les Boucliers de Quetzalcoatl`
So all the titles listed in the code, as well as this one, only contain ASCII characters. So maybe it does not matter.

Reply:
As far as I can tell, `setWindowCaption` was specified to use ASCII in 188cdf937d17c27ddd221178dcf9a0da7898b035, and was changed to use ISO-8859-1 in bb28ed7b7a10535c4f3f275b3bef3643f9ce46d3, while the relevant code in the Sword2 engine predates both of those commits. As such, I would assume that the caption used here is always ASCII. Having said that though, it might be worth removing the use of `setWindowCaption` both here and in the icb engine, since those are the only two engines to use a custom window caption, and there doesn't seem to be much of an advantage to this compared to using the standard caption.

Reply:
Yes, I is probably fine to assume ASCII here since that is what was initially assumed.
Or indeed we could just remove this code and use the standard caption.